cmk/gui/plugins/wato/check_parameters/fs_mount_options.py

```diff
@@ -19,14 +21,36 @@
 
 
 def _parameter_valuespec_fs_mount_options():
-    return ListOfStrings(
+    expected = ListOfStrings(
         title=_("Expected mount options"),
         help=_("Specify all expected mount options here. If the list of "
                "actually found options differs from this list, the check will go "
                "warning or critical. Just the option <tt>commit</tt> is being "
                "ignored since it is modified by the power saving algorithms."),
         valuespec=TextUnicode(),
     )
+    ignore = ListOfStrings(
+        title=_("Mount options to ignore"),
+        help=_("Specify all mount options that should be ignored when inspecting "
+               "the list of actually found options. The options <tt>commit</tt>, "
+               "<tt>localalloc</tt>, <tt>subvol</tt>, <tt>subvolid</tt> are "
+               "ignored by default."),
+        valuespec=TextUnicode(),
+        default_value = ["commit=", "localalloc=", "subvol=", "subvolid="],
```

Comment:
I suggest to use `"commit"` etc without the trailing "=". That should be validated using the input validation function supplied via `validate=` kwarg.